openreview/venue_request/process/matching_pre_process.py

```diff
@@ -1,4 +1,20 @@
 def process(client, note, invitation):
+    _, baseurl_v2 = openreview.tools.get_base_urls(client)
+    client_v2 = openreview.api.OpenReviewClient(baseurl=baseurl_v2, token=client.token)
+    forum_note = client.get_note(note.forum)
+    venue_id = forum_note.content.get('venue_id', '')
+    domain = client_v2.get_group(venue_id)
+    compute_affinity_scores = note.content.get('compute_affinity_scores') != 'No'
 
-    if 'Yes' in note.content['compute_affinity_scores'] and 'upload_affinity_scores' in note.content:
-        raise openreview.OpenReviewException('Either upload your own affinity scores or select affinity scores computed by OpenReview')
\ No newline at end of file
+    if compute_affinity_scores and 'upload_affinity_scores' in note.content:
+        raise openreview.OpenReviewException('Either upload your own affinity scores or select affinity scores computed by OpenReview')
+
+    if forum_note.content.get('api_version', '1') == '2':
+        matching_group = note.content['matching_group']
+        senior_area_chairs_name = domain.get_content_value('senior_area_chairs_name')
+        if senior_area_chairs_name and matching_group.endswith(senior_area_chairs_name): return
```

Comment:
There is no Python convention about this but our code put the `return` statement in a new line. We would like to keep consistency in the code style.